capstone/src/main/java/com/google/sps/data/StoreRestaurantDataHelper.java

```diff
@@ -0,0 +1,47 @@
+package com.google.sps.data;
```

Comment:
do you think we could name this something more idiomatic, like say businessDataFetcher or businessDataWriter or something? alternatively if you wanted to abstract it I'd recommend using Utils in the name, so businessDataUtils or somesuch.

Reply:
I will do businessDataUtils for greater flexibility